client/js/character.js

```diff
@@ -86,16 +86,19 @@ define(['entity', 'transition', 'timer'], function(Entity, Transition, Timer) {
             this.flipSpriteY = false;
 
             if(_.indexOf(oriented, animation) >= 0) {
+                if(o === Types.Orientations.LEFT) {
+                    this.flipSpriteX = true
```

Comment:
I've noticed there's a semicolon missing, the OG code missed it as well

Reply:
They are optional in Js, but I'll add it.
It might be good to have some code-styling tooling setup in the repository, to make sure everything is always nicely formatted.